.github/workflows/delete-review-app.yml

```diff
@@ -47,6 +47,11 @@ jobs:
         with:
           terraform_version: ${{ env.TERRAFORM_VERSION }}
 
+      - uses: DFE-Digital/github-actions/set-arm-environment-variables@master
+        if: env.TF_STATE_EXISTS == 'true'
+        with:
+          azure-credentials: ${{ secrets.AZURE_CREDENTIALS_REVIEW_AKS }}
```

Comment:
New code has _AKS?

Reply:
I'll change the secret refs in the next PR

Reply:
decided to update in this PR, changes pushed